scripts/ci/verify_linter.py

```diff
@@ -0,0 +1,208 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+This script is used to verify linter on extensions.
+
+It's only working on ADO by default. If want to run locally,
+please update the target branch/commit to find diff in function find_modified_files_against_master_branch()
+"""
+
+import os
+import json
+from subprocess import check_output, check_call
+from pkg_resources import parse_version
+
+
+def separator_line():
+    print('-' * 100)
+
+
+class ModifiedFilesNotAllowedError(Exception):
+    """
+    Exception raise for the scenario that modified files is conflict against publish requirement.
+    Scenario 1: if modified files contain only src/index.json, don't raise
+    Scenario 2: if modified files contain not only extension code but also src/index.json, raise.
+    Scenario 3: if modified files don't contain src/index.json, don't raise.
+    """
+
+    def __str__(self):
+        msg = """
+        ---------------------------------------------------------------------------------------------------------
+        You have modified both source code and src/index.json!
+
+        There is a release pipeline will help you to build, upload and publish your extension.
+        Once your PR is merged into master branch, a new PR will be created to update src/index.json automatically.
+
+        If you want us to help to build, upload and publish your extension, src/index.json must not be modified.
+        ---------------------------------------------------------------------------------------------------------
+        """
+        return msg
+
+
+class AzExtension:
+    def __init__(self, extension_name):
+        self.extension_name = extension_name
+
+    @staticmethod
+    def _cmd(cmd):
+        print(cmd)
+        check_call(cmd.split(), shell=True)
+
+    def add_from_url(self, url):
+        self._cmd('az extension add -s {} -y'.format(url))
+
+    def remove(self):
+        self._cmd('az extension remove -n {}'.format(self.extension_name))
+
```

Comment:
Can we just add it here?
```
def linter(self):
        self._cmd('azdev linter --include-whl-extensions {}'.format(self.extension_name))
```

Reply:
But it's not belong to `az`.

Reply:
Well. I figure out a way, Renmae the class to `Azdev` and method name prefix with `extension`, like `extension_add`, `extension_remove`, but `linter(self)` only.

What do you think?

Reply:
Append `Helper` to the class name.